schemas/oic.sec.didtype.json

```diff
@@ -16,18 +14,5 @@
           },
           "required": ["uuid"]
```

Comment:
Yeah - that's the fun of defining the schema in the same language it's attempting to define.

The uuid definition above basically says that "the property called `uuid` can have a value of type `string` which has to match the pattern `^([0-9]{4})-(1[0-2]|0[1-9])-(3[0-1]|2[0-9]|1[0-9]|0[1-9])$`. (taken from the definition of `uuid` in oic.types-schema.json)